src/main/webapp/i18n/en/exercise.json

```diff
@@ -219,7 +219,8 @@
                     "pattern": {
                         "JAVA": "The package name must consist of one or more valid Java identifiers separated by '.', e.g. \"net.java\"!",
                         "KOTLIN": "The package name must consist of one or more valid Kotlin identifiers separated by '.', e.g. \"net.kotlin\"!",
-                        "SWIFT": "The package name must consist of one or more valid Swift identifiers without any separator, e.g. \"SwiftEx\"!"
+                        "SWIFT": "The package name must consist of one or more valid Swift identifiers without any separator, e.g. \"SwiftEx\"!",
+                        "JAVA_BLACKBOX": "The package name must be a valid Java identifier. In addition, no dots are allowed for the blackbox project type!"
```

Comment:
What I meant with that was: For all other project types, dots are usually allowed as separators in the package name, but in the specific case of the blackbox type, they aren't. I guess `In addition, no dots are allowed in the package name of blackbox projects!` would make it more clear?

Reply:
I’d prefer the ‘for the blackbox project type’ as-is. The UI already calls the choice Gradle/Maven/Dejagnu the ‘Project Type’.

Follow-up question: Should that then refer to ‘DejaGnu’ tests in the warning message rather than blackbox-tests so that the user can mentally connect those two things? Otherwise they might now know that DejaGnu = Blackbox.

Reply:
`In addition, no dots are allowed in the package name of blackbox projects!` would make it very clear :+1:

Reply:
But it's optinal, I now understand it. I agree with @b-fein, writing DejaGnu would make sense since the project type is called that

Reply:
```suggestion
                        "JAVA_BLACKBOX": "The package name must be a valid Java identifier. In addition, no dots are allowed in the package name of DejaGnu projects"
```